src/commands/utility/ServerInfoCommand.js

```diff
@@ -13,8 +13,10 @@ class ServerInfoCommand extends Command{
     async execute() {
         const guild = this.message.guild;
         
+        let owner = await this.bot.users.fetch(guild.ownerID);
+      
         let generic = '';
-        generic += `**Owner:** <@!${guild.ownerID}> \n`;
+        generic += `**Owner:** <@!${guild.ownerID}> (${owner.user.username.tag}) \n`;
```

Comment:
```suggestion
        generic += `**Owner:** <@!${guild.ownerID}> (${owner.tag}) \n`;
```
owner is already of type User and not GuildMember and you just added the tag instead of replacing the username with it